Dockerfile

```diff
@@ -82,12 +83,14 @@ RUN addgroup -S -g $APP_GID $APP_USER \
 	&& rm -rf /var/cache/apk/*
 
 COPY --from=builder2 /app $APP_HOME
+RUN chown -R $APP_UID:$APP_GID $APP_HOME/arranger-admin
 
 WORKDIR $APP_HOME
 
 USER $APP_USER
 
-CMD envsubst '$PORT,$REACT_APP_ARRANGER_ADMIN_ROOT' < /etc/nginx/nginx.conf.template > $NGINX_CONF_PATH && exec nginx -c $NGINX_CONF_PATH -g 'daemon off;'
+CMD envsubst < docker/ui/env-config.template.js > /app/arranger-admin/env-config.js \
```

Comment:
Grabs the process environment and replaces values in a file that will be loaded by the browser.